src/Google.Storage.V1/StorageClient.cs

```diff
@@ -73,7 +74,17 @@ public abstract partial class StorageClient
         /// <returns>The created <see cref="StorageClient"/>.</returns>
         public static StorageClient Create(GoogleCredential credential = null)
         {
-            credential = credential ?? Task.Run(() => GoogleCredential.GetApplicationDefaultAsync()).Result;
+            try
+            {
+                credential = credential ?? Task.Run(() => GoogleCredential.GetApplicationDefaultAsync()).Result;
+            }
+            catch (AggregateException e)
+            {
+                // Unwrap the first exception, a bit like await would.
+                // It's very unlikely that we'd ever see an AggregateException without an inner exceptions,
+                // but let's handle it relatively gracefully.
+                throw e.InnerExceptions.FirstOrDefault() ?? e;
```

Comment:
1. No, I think it's okay just to rethrow the first. Again, this matches the await behaviour.

2. Theoretically nicer, but so improbable that we'll ever get there that it's not worth it IMO.